jestconfig.json

```diff
@@ -12,5 +12,13 @@
         "jsx",
         "json",
         "node"
+    ],
+    "globals": {
+        "ts-jest": {
+          "compiler": "ttypescript"
```

Comment:
Is the `ttypescript` correct?

Reply:
Yes, it's required by ts-auto-mock https://typescript-tdd.github.io/ts-auto-mock/installation#packagejson--jestconfigjs-without-the-jest-scope-1